splinterd/src/config/mod.rs

```diff
@@ -807,6 +809,17 @@ impl Config {
             debug!("Config: whitelist: {:?} (source: {:?})", list, source,);
         }
     }
+
+    pub fn verify_files(&self) -> Result<(), ConfigError> {
```

Comment:
This check should be done outside of the config module, as it has nothing to do with verifying the validity of the configuration, but rather the validity of the environment. In other words, the config module's primary concern is config data, but no how that data is applied.